api/swagger.yaml

```diff
@@ -9202,6 +9202,15 @@ paths:
           description: "Image name or ID"
           type: "string"
           required: true
+        - name: "platform"
+          type: "string"
+          in: "query"
+          description: |
+            JSON encoded OCI platform describing platform to show the history for.
+            If not provided, the host platform will be used. If it's not
+            available, any present platform will be picked.
+
+            Example: {"os": "linux", "architecture": "arm", "variant": "v5"}
```

Comment:
Non-blocking; wondering if it looks better with the example in back-ticks;

Without:

<img width="952" alt="Screenshot 2024-09-11 at 17 23 09" src="https://github.com/user-attachments/assets/99b901c2-dae7-49cb-8114-4b26ea09fb56">


With:

<img width="934" alt="Screenshot 2024-09-11 at 17 22 26" src="https://github.com/user-attachments/assets/538e85e1-6e68-44dc-8c2c-7c9fe98429f1">

Reply:
Done!